README.md

```diff
@@ -76,25 +77,29 @@ Beginning March 2020, CoinMarketCap requires an API key to access currency marke
+| coinmarketcap     | _String_               | (unprotected API key)                                           | [API key][55] to use when fetching current market price data. (Use this if you stop seeing price data)                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| gasPrice          | _Number_               | (varies)                                                        | Denominated in `gwei`. Default is loaded at runtime from the `eth gas station` api                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| token             | _String_               | 'ETH'                                                           | The reference token for gas price                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| gasPriceApi       | _String_               | [ethgasstation](https://ethgasstation.info/json/ethgasAPI.json) | The API endpoint to retrieve the gas price. Could be also [BSC](https://api.bscscan.com/api?module=proxy&action=eth_gasPrice&apikey=YourApiKeyToken), [Polygon](https://api.polygonscan.com/api?module=proxy&action=eth_gasPrice&apikey=YourApiKeyToken), [Avalanche](https://api.snowtrace.io/api?module=proxy&action=eth_gasPrice&apikey=YourApiKeyToken), [Heco](https://api.hecoinfo.com/api?module=proxy&action=eth_gasPrice&apikey=YourApiKeyToken), [Moonriver](https://api-moonriver.moonscan.io/api?module=proxy&action=eth_gasPrice&apikey=YourApiKeyToken), |
```

Comment:
The default in this PR looks like it's no longer ethGasStation - it's an eth_gasPrice call to etherscan?

(Thanks for all these links! Great.)

Reply:
Would you consider adding a table below the config options with example tokens and gasPriceApi urls? If we can reliably fetch this data without an api key maybe that part of the url could be omitted?

### Example `token` and `gasPriceApi` configurations

Any network call which returns a JSON-RPC response formatted as below is supported:
```
{"jsonrpc":"2.0","id":73,"result":"0x6fc23ac00"}
```
(By default, reporter calls `https://api-cn.etherscan.com` for the Ethereum network gas price.)  

Other networks include:
| Network | `token` | `gasPriceApi` |
| ----- | ----- | ----- |
| Binance | BNB | https://api.bscscan.com/api?module=proxy&action=eth_gasPrice |
| Polygon | MATIC | https://api.polygonscan.com/api?module=proxy&action=eth_gasPrice |
| Avalanche | AVAX | https://api.snowtrace.io/api?module=proxy&action=eth_gasPrice |
| Heco | BIRD | https://api.hecoinfo.com/api?module=proxy&action=eth_gasPrice |
| Moonriver | MOVR | https://api-moonriver.moonscan.io/api?module=proxy&action=eth_gasPrice |

Reply:
Makes sense, I made this change on the doc and config files.